src/doc/unstable-book/src/SUMMARY.md

```diff
@@ -199,7 +199,6 @@
     - [sort_internals](library-features/sort-internals.md)
     - [sort_unstable](library-features/sort-unstable.md)
     - [splice](library-features/splice.md)
-    - [step_by](library-features/step-by.md)
```

Comment:
Hm... The `iterator_step_by` entry is missing from this list. Do you mind adding it, please? 😃